sensor_msgs/NavSatFix waypoint
---
int64 status